lib/tic_tac_toe_rules.rb

```diff
@@ -0,0 +1,40 @@
+class TicTacToeRules
+    def winner?(player_stats)
+        return false if player_stats.length <= 2 
+        evaluate_player_stats(player_stats)
+    end
+
+    def evaluate_player_stats(player_stats)
```

Comment:
Yes, I'm not satisfied with the name, but since I'm using it in two methods, I thought this name would be a compromise. I was trying to stay DRY. This is my attempt to create a method that looks at the player's moves and matches it to each set of winning combo sets.